index.html

```diff
@@ -0,0 +1,98 @@
+            <div id="col-2">
+                <div id="content-top">
+                    <div>
+                        <img src="./img/intuit_logo.jpg" alt="intuit" id="logo">
+                    </div>
+                    <div id="items">
+                        <ul>
+                            <li><a href="#"> <img src="./img/mint.jpg" alt=""></a> </li>
+                            <li><a href="#"><img src="./img/quickboks.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/turbotax.jpg" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div id="form-container">
+                    <div>
+                        <div>
+                            <h1>Create an Intuit account</h1>
+                        </div>
+                        <div>
+                            <p id="txt-1">One account for averything Intuit, Including Ming.
+                        </div>
+                        <div>
+                            <a id="a-1" href="#">Learn more</a></p>
+                        </div>
+
+
+                    </div>
+
+                    <div>
+                        <form idaction="">
+                            <div>
+                                <label for="emailUser" id="input-lb-email">Email address</label>
+                                <div id="input-email">
+                                    <input type="text">
+                                </div>
+
+                            </div>
+                            <div>
+                                <label for="phone" id="input-lb-phone">Phone (recommended)</label>
+                                <div id="input-phone">
+                                    <input type="text">
+                                </div>
+                            </div>
+                            <div>
+                                <label for="password" id="input-lb-pass">Password</label>
+                                <div id="input-pass">
+                                    <input type="text">
+                                </div>
+                            </div>
+                            <div id="div-btn-form">
+                                <button id="btn-form">Create Account</button>
+                            </div>
+                            <em>
+                                <p id="p-agree">By clicking Sign in, you agree to our <a href="#">terms</a>
+                                    and have read and acknowlegement our <a href="">Privacy Statement.</a></p>
+                            </em>
+                        </form>
+                    </div>
+                    <div id="footer-form">
+                        <p>Invisible reCAPTCHA by Google <a href=""> Privacy Policy</a> and <a href=""> Terms of
+                                Use</a>.</p>
+                    </div>
+                </div>
+                <div id="footer">
```

Comment:
`OPTIONAL`

Consider using `<footer>` element for the **footer content**, this helps to communicate meaning to the browser, developer, reader, and any other technologies interpreting the document (e.g. voice assistants). :ok_hand: